source/_integrations/zwave_js.markdown

```diff
@@ -450,11 +450,11 @@ These are notification events fired by devices using the Notification command cl
 }
 ```
 
-#### Multilevel Switch Command Class
+#### Multilevel switch command class
 
-These are notification events fired by devices using the Multilevel Switch command class. There are events for start level change and stop level change. These would typically be used in a device like the Aeotec Nano Dimmer with an external switch to respond to long button presses.
+These are notification events fired by devices using the multilevel switch command class. There are events for start level change and stop level change. These would typically be used in a device like the Aeotec Nano Dimmer with an external switch to respond to long button presses.
```

Comment:
Multilevel Switch is a name of a command class. That is a bit lost now.

Reply:
good catch, I will do a quick sweep